### PR TITLE
[DDW-1073] Daedalus copy updates for Incentivized Testnet - Balance check

### PR DESCRIPTION
**Pull request by tomothespian**
This PR introduces Daedalus copy updates for ITN - Balance check, Legacy wallet delete description and disabling "Wallet Import" feature

## Todos

- [x] Disable wallet import
- [x] Add Legacy wallet delete description

## Screenshots

- Screenshot

_Add Walled_
EN - Add Walled

JA - Add Walled

_Disabled import feature_

EN - Disabled import feature

Screen Shot 2019-11-13 at 20 27 32

_Notification - Move Testnet ADA_

EN - Notification - Move Testnet ADA

JA - Notification - Move Testnet ADA

_Notification - NO Rewards Wallet_

EN - Notification - NO Rewards Wallet

JA - Notification - NO Rewards Wallet

_Restore Wallet TAB 1_

EN - Restore Wallet TAB 1

JA - Restore Wallet TAB 1

_Restore Wallet TAB 2_

EN - Restore Wallet TAB 2

JA - Restore Wallet TAB 2

_Transfer Funds dialog - step 1_

EN - Transfer Funds dialog - step 1

JA - Transfer Funds dialog - step 1

_Deletion warning message on Legacy Wallet_

EN- Deletion warning message on Legacy Walet

Screen Shot 2019-11-13 at 20 26 24

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1573673584492700)
- [ ] Copy updates must match: https://docs.google.com/document/d/1NR_cA7SRNCOX1PV7-8tLKGeQsptC0YhhkuRYB2lyvXM/edit
- [ ] Check if "Import Wallet" feature is disabled on ITN
- [ ] Delete legacy wallet (Wallet Settings screen) should have a deletion warning message.

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: source/renderer/app/i18n/locales/en-US.json [hunk 1]
```diff
@@ -428,15 +428,15 @@
   "test.environment.selfnodeLabel": "!!!Selfnode vx",
   "test.environment.stagingLabel": "Staging",
   "test.environment.testnetLabel": "Testnet",
-  "wallet.add.dialog.create.description": "Create a new wallet",
+  "wallet.add.dialog.create.description": "Create a wallet for the balance check",
```
Comment (darko-mijic) [inline source/renderer/app/i18n/locales/en-US.json]: Revert this line to what it was @tomothespian

Comment (tomothespian) [reply source/renderer/app/i18n/locales/en-US.json]: @darko-mijic Done!

**Comment by a-rukin** [comment]
image

Now with this copy change radio buttons look too separated, so I'd go without sticking second one to the right, just add 40px padding and all good here.
cc @nikolaglumac @darko-mijic

**Comment by a-rukin** [comment]
image

Content looks shifted to the bottom 🤔

**Comment by daniloprates** [comment]
> Now with this copy change radio buttons look too separated
> Content looks shifted to the bottom

@a-rukin [done](https://github.com/input-output-hk/daedalus/pull/1680/commits/c3839c26966e45ff41f7d7434c66e9abaa2afaa6).

**Comment by IuliaDolishniak** [comment]
@tomothespian I am not sure if this is part of this PR but Password is requested to be 7 characters long but only 10 are accepted

**Comment by DmitriiGaico** [comment]
@tomothespian HiTomislav!
Could you please check where is the ITC theme in StoryBook?
image

**Comment by tomothespian** [comment]
> @tomothespian I am not sure if this is part of this PR but Password is requested to be 7 characters long but only 10 are accepted

@IuliaDolishniak  This one is done!

**Comment by daniloprates** [comment]
> Could you please check where is the ITN theme in StoryBook?

@DmitriiGaico [fixed](https://github.com/input-output-hk/daedalus/pull/1680/commits/f19e13391311a5b683440b39abbf0d3131f2c112).

**Pull request merged**